triangle/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: eolianoe <eolianoe At GoogleMAIL DoT com>
+# Maintainer: A Austin <acm.austin at gmail.com>
+# Contributor: Martin Ortbauer <mortbauer@gmail.com>
+
+pkgbase=archimedes-tools
+pkgname=('triangle' 'showme')
+pkgver=1.6
+pkgrel=7
+arch=('i686' 'x86_64')
+url='http://www.cs.cmu.edu/~quake/archimedes.html'
+pkgdesc='Archimedes is a set of tools, including mesh generators, for performing unstructured finite element simulations'
+license=('custom')
+source=('http://ftp.mcs.anl.gov/pub/petsc/externalpackages/Triangle.tar.gz')
```

Comment:
If you could keep the original source and make a patch on top of that instead of change the source entirely that would be much appreciated.

Reply:
Also if you coulde add `${pkgname}-${pkgver}::` before the url that would also be nice. See the warning here: https://wiki.archlinux.org/index.php/PKGBUILD#source